visualization/app/codeCharta/ui/nodeContextMenu/nodeContextMenu.component.html

```diff
@@ -7,6 +7,7 @@
 				ng-disabled="primary === color && !isPrimary"
 				ng-click="$ctrl.clickColor(color)"
 				class="colorButton"
+				title="Color folder"
```

Comment:
Is this a "color folder" or should it indicate to "colorize the folder"? I am not sure where this is used.

Reply:
colorize. fixing